src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -547,6 +550,30 @@ private IEnumerable<DomainError> BuildLastNameValidationFailures(DraftApprentice
             }
         }
 
+        private IEnumerable<DomainError> BuildEmailValidationFailures(DraftApprenticeshipDetails draftApprenticeshipDetails)
+        {
+            bool EmailIsValid(string email)
+            {
+                try
+                {
+                    var _ = new MailAddress(email);
+                    return true;
+                }
+                catch
+                {
+                    return false;
+                }
+            }
+
+            if (draftApprenticeshipDetails.Email != null)
```

Comment:
This works, but just a personal opinion, i usually prefer IsNullEmptyOrWhiteSpace instead of null,

Reply:
I know what you mean, but if the email has any value in it then it has to be validated, I just don't want whitespace ending up in the DB

Reply:
Makes perfect sense, I read the code incorrectly :)